test/cpp/jit/test_alias_analysis.cpp

```diff
@@ -9,10 +9,8 @@
 namespace torch {
 namespace jit {
 
-inline c10::OperatorOptions aliasAnalysisFromSchema() {
-  c10::OperatorOptions result;
-  result.setAliasAnalysis(c10::AliasAnalysisKind::FROM_SCHEMA);
-  return result;
+inline c10::AliasAnalysisKind aliasAnalysisFromSchema() {
```

Comment:
nit: doensn't look like we'd need to factor this out into a separate function anymore. Same for several similar functions below in this changeset.

Reply:
Yeah, I saw this too, chose not to inline to reduce footprint of this patch